addons/mail/static/src/core/web/mail_composer_chatgpt.js

```diff
@@ -1,42 +1,46 @@
-import { ChatGPTPromptDialog } from "@html_editor/main/chatgpt/chatgpt_prompt_dialog";
-
-import { htmlJoin } from "@mail/utils/common/html";
-
-import { Component, markup } from "@odoo/owl";
-
 import { _t } from "@web/core/l10n/translation";
 import { registry } from "@web/core/registry";
 import { standardFieldProps } from "@web/views/fields/standard_field_props";
 
+import { htmlJoin } from "@mail/utils/common/html";
+
+import { Component, markup, onMounted, onWillUnmount } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+
 export class MailComposerChatGPT extends Component {
     static template = "mail.MailComposerChatGPT";
     static props = { ...standardFieldProps };
 
     setup() {
-        this.btnLabel = _t("AI"); // workaround to translate short string
+        this.store = useService("mail.store");
+        this.orm = useService("orm");
+        this.aiChatLauncher = useService("aiChatLauncher");
```

Comment:
you cannot have code dependent on ai in mail/. I see that ai_apps depends on mail, so either reverse that dependency, or ai code should stay in ai_apps addon

Reply:
I moved the mail_composer_chatgpt to the ai_apps addon